package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cowswap",
-  "version": "1.40.0",
+  "version": "1.40.1",
```

Comment:
Please, don't change the version manually.
Read more: https://www.notion.so/cownation/Deploy-Web-CoW-Swap-Explorer-2a38ae63426148048ec8a898d6a94c4e